examples/create-token-bridge-eth/index.ts

```diff
@@ -84,6 +86,41 @@ async function main() {
   );
 
   console.log(`Deployed in ${getBlockExplorerUrl(parentChain)}/tx/${txReceipt.transactionHash}`);
+
+  // waiting for retryables to execute, to prevent race conditions
+  await txReceipt.waitForRetryables({ orbitPublicClient: childChainPublicClient });
```

Comment:
this should be sufficient most of the time, but in an edge case where the 2nd retryable is executed (manually) before the 1st, they would both be appeared to executed although the token bridge deployment would fail

we can also check for contract existence